examples/chart/teleport-demo/docker/namespace-cleaner/rootfs/scripts/namespace-cleaner.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# This script takes a space-separated list of namespaces which should be deleted by kubectl after the helm chart is gone
+set -e
+
+function namespacecleaner_log() {
```

Comment:
Nitpick: I am not sure all that log helpers are relevant for K8s. You anyway will look at specific pod name logs and from the name you will know for what this log is.